src/main/java/com/gsheet/student/controller/StudentController.java

```diff
@@ -0,0 +1,63 @@
+package com.gsheet.student.controller;
+
+import com.gsheet.student.dto.StudentDto;
+import com.gsheet.student.entity.Student;
+import com.gsheet.student.service.StudentService;
+import com.gsheet.student.transformer.StudentToStudentDtoTransformer;
+import lombok.RequiredArgsConstructor;
+import lombok.SneakyThrows;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.*;
+
+import java.util.List;
+
+@RestController
+@RequestMapping("/api/students")
+@RequiredArgsConstructor
+public class StudentController {
+
+    @Autowired
+    private StudentService studentService;
+
+    @Autowired
+    private StudentToStudentDtoTransformer studentToStudentDtoTransformer;
+
+    @SneakyThrows
```

Comment:
We don't use `@SneakyThrows`, when we are talking about exception strategy I prefer to see all logic on the code related to it throw and catch